worker/peergrouper/desired.go

```diff
@@ -35,14 +36,19 @@ func desiredPeerGroup(info *peerGroupInfo) ([]replicaset.Member, map[*machineTra
 	}
 	changed := false
 	members, extra, maxId := info.membersMap()
-	logger.Debugf("calculating desired peer group")
```

Comment:
put all the logging into a single logging call, instead of lots of bits and pieces.
make the formatting for "extra" more similar to the formatting for the rest, rather than a bunch of attributes that don't make sense.